Libraries/Components/Slider/Slider.js

```diff
@@ -127,6 +127,32 @@ var Slider = React.createClass({
      * Used to locate this view in UI automation tests.
      */
     testID: PropTypes.string,
+
+    /**
+     * The color used for the progress
+     *
+     * *using in android >= 21*
+     * @platform android
+     */
+    progressColor: PropTypes.string,
```

Comment:
I think we should just use `maximumTrackTintColor` and `minimumTrackTintColor` props since it does the same thing as those ios props, this will make it easier to use for cross platform apps.